app/models/miq_search.rb

```diff
@@ -41,6 +41,21 @@ def self.visible_to_all
     where("search_type=? or (search_type=? and (search_key is null or search_key<>?))", "global", "default", "_hidden_")
   end
 
+  def self.visible_to_current_user
+    where(:search_type => 'user', :search_key => User.current_user.userid)
+  end
+
+  def self.filters_by_type(type)
+    case type
+    when "global" # Global filters
```

Comment:
While I feel `"global"` and `"my"` are ui strings, I notice that this file is littered with them.

Do we also need to include `"default"` here? (No opinion in this comment. I noticed it elsewhere in this file and was wondering)